src/App.tsx

```diff
@@ -8,7 +8,39 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { СompletedStatus } from './types/СompletedStatus';
```

Comment:
There seems to be a typo in the import statement for `СompletedStatus`. Ensure that the file name and path are correct. The character 'С' might be a Cyrillic letter instead of the Latin 'C'.